integration_test/third_party_apps_data/applications/hbase/centos_rhel/install

```diff
@@ -1,11 +1,13 @@
 set -e
 
+HBASE_VERSION=$(curl https://apache.mirror.gtcomm.net/hbase/stable/ | grep --extended-regexp 'hbase-[0-9.]+-bin.tar.gz' | grep --extended-regexp --only-matching '[0-9]+\.[0-9]+\.[0-9]+' | head -1)
```

Comment:
I think --only-matching makes sense for the first grep command as well, in case there happen to be two version number-looking things on that line.